src/v3/src/transformer/redirect/redirectTransformer.test.ts

```diff
@@ -84,8 +85,8 @@ describe('Success Redirect Transform Tests', () => {
     );
 
     expect(formBag.uischema.elements.length).toBe(3);
-    expect(formBag.uischema.elements[0].type).toBe('Description');
-    expect((formBag.uischema.elements[0] as DescriptionElement).options?.content)
+    expect(formBag.uischema.elements[0].type).toBe('ActionPending');
```

Comment:
nit: I'm seeing mixed use of the custom matcher vs built-in matcher: `expect(x).toBeElementType('value')` vs `expect(x.type).toBe('Value')`, IMHO it implies the custom matcher isn't doing enough work to be worthwhile.

Reply:
Yeah, I was going through and didn't migrate all of them, but I realized that we shouldn't using the `as` cast  (it never fails, even if the element properties are invalid for the type) but instead use a different kind of type check. I agree that the `toBeElementType` isn't providing that rigor anyways so I can remove it.